TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -887,5 +895,19 @@ private void WritePeakQuantificationResultsToTsv(FlashLFQResults flashLFQResults
 
             FinishedWritingFile(peaksPath, nestedIds);
         }
+
+        private List<PeptideSpectralMatch> FilterPSMList(PostSearchAnalysisParameters parameters, List<PeptideSpectralMatch> psms)
```

Comment:
rename to `FilterPsmList` (Pascal case)

Reply:
this method doesn't need any parameters, you can reference both of these objects through `this.Parameters`. alternatively you could make the method `static` and only require the `PostSearchAnalysisParameters` parameter